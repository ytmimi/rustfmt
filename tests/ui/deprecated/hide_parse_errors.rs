// rustfmt-hide_parse_errors: true
