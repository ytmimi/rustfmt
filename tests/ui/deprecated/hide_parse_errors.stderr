Warning: the `hide_parse_errors` option is deprecated. Use `show_parse_errors` instead
